ENGAGEHF/Medications/MedicationsLegendEntry.swift

```diff
@@ -0,0 +1,21 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct MedicationsLegendEntry: Hashable, Equatable {
+    let color: Color
+    let description: String
+    
+    
+    init(for type: MedicationRecommendationType) {
+        self.color = RecommendationSymbolColor.color(for: type)
```

Comment:
What do you think about making this a computed properties of the `MedicationRecommendationType`?